api/app/src/external/commonwell/link/shared.ts

```diff
@@ -0,0 +1,22 @@
+import { Person } from "@metriport/commonwell-sdk";
+import { apiUrl } from "../api";
+
+import { OID_PREFIX } from "../../../shared/oid";
+
+export const createReferenceLink = (patientId: string, orgId: string) => {
+  return `${apiUrl}/v1/org/${OID_PREFIX}${orgId}/patient/${patientId}`;
```

Comment:
So i went with the just using the in-memory because the url seemed pretty straightforward once i added the patientId. I do agree about the commonwell-sdk point so i can update that